pom.xml

```diff
@@ -189,6 +199,13 @@
           <source>${java.version}</source>
           <target>${java.version}</target>
         </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.ow2.asm</groupId>
```

Comment:
@spulci same here.. do we need this? where is this being used?

Reply:
Removed...was a cut&paste error on my side sorry.